ingest/example/bulk_pretty

```diff
@@ -0,0 +1,27 @@
+{  
+  "index":{  
+    "_id":"1",
+    "_index":"dtdi",
+    "_type":"mineral-observation"
+  }
+}{  
+  "locality":[  
+    {  
+      "country":"Canada",
+      "region":"Québec, Canada",
+      "mine":"Poudrette quarry (Demix quarry; Uni-Mix quarry; Desourdy quarry; Carrière Mont Saint-Hilaire), Mont Saint-Hilaire, La Vallée-du-Richelieu RCM, Montérégie, Québec, Canada",
+      "coordinates":{  
+        "lat":"45.56277778",
+        "lon":"-73.14166667"
+      },
+      "mindat":"http://www.mindat.org/loc-599.html"
+    }
+  ],
+  "description":"Named after the Abenaki, the indigenous people who inhabited the area around Mont Saint-Hilaire in southern Québec, prior to the arrival of European settlers.",
+  "mindat":"http://www.mindat.org/min-2.html",
+  "mineral":{  
```

Comment:
Should we have max age and min age for the observed mineral specimen?  (see #4, and ask @xgmachina about it)